docs/stream/introduction.mdx

```diff
@@ -0,0 +1,343 @@
+---
+title: "Introduction"
+---
+
+import CodeTabs from "@theme/CodeTabs";
+import TabItem from "@theme/TabItem";
+
+# Svix Stream
+
+Svix Stream is an event streaming platform that enables services to reliably deliver events to multiple destinations through a single integrated API.
```

Comment:
I don't explains what this is clearly enough for someone that has no idea what this is. Remember that we are inventing something new here, so a more clear explanation of what it is and why it's useful will probably go a long way.

I can try and come up with concrete suggestions if you want to brainstrom together, but I think I'd at least mention "sending a stream of events to your customers", and give a few examples.
I'd also compare to Svix webhooks and explain how it's different.

Reply:
I think also drawing parallels to thing they may know can help in a second sentence. E.g. "You can think of it like Kafka + Bento, but self-serve and across the internet rather than within the same company."